blivet/util.py

```diff
@@ -714,6 +714,17 @@ def compare(first, second):
     else:
         return (first > second) - (first < second)
 
+
+def dedup_list(alist):
```

Comment:
I think just "dedup" itself is fine but I think there should b e a docstring noting how the list deduplication works as I can imagine multiple ways how it can be done (keeping order, not keeping order, deduplicating only element sequences, etc.).